src/lib/hasExample.ts

```
@@ -2,13 +1,26 @@
+export function hasExample(schema: any, visited: Set<any> = new Set(), level: number = 0): boolean {
+  if (visited.has(schema)) {
+    if (level > 10) {
+      return false // Assume no example for circular references.
+    }
+
+    visited = new Set()
```

Biome

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)